### PR TITLE
masonry: Disable render tests by default

### PR DESCRIPTION
**Pull request by waywardmonkeys**
Set both `ENABLE_RENDER_TESTS=1` and `ENABLE_RENDER_SNAPSHOTS=1` in your environment to run the tests.

### CODE REVIEW

**Comment by waywardmonkeys** [comment]
My concerns with this PR:

1) There's no documentation for how to run the tests yet, so nowhere to mention how to re-enable them.
2) I'm not sure we need 2 separate environment variables for now.

File: crates/masonry/src/testing/harness.rs [hunk 1]
```diff
@@ -238,7 +243,7 @@ impl TestHarness {
     /// Create a bitmap (an array of pixels), paint the window and return the bitmap as an 8-bits-per-channel RGB image.
     pub fn render(&mut self) -> RgbaImage {
         let (scene, _tree_update) = self.render_root.redraw();
-        if std::env::var("SKIP_RENDER_TESTS").is_ok_and(|it| !it.is_empty()) {
+        if !check_env_enabled("ENABLE_RENDER_TESTS") {
```
Comment (DJMcNab) [inline crates/masonry/src/testing/harness.rs:246]: So as it happens, the render tests mostly work fine

So I'd probably revert this half of the change

**Comment by waywardmonkeys** [comment]
One other difference is that one part of this was reading the env variable at runtime via `std::env::var` while the other part was reading it at compile time via `option_env!`.

**Comment by DJMcNab** [comment]
So, my expected updates for this PR would be.
1) Keep the change to read at runtime for both
2) Re-invert the `SKIP_RENDER_TESTS` flag, as that is *only* useful for Windows CI

Alternatively, we could close this pending the fixed snapshot tests (i.e. #233), but that is at least another couple of weeks away.
I can make these changes if needed.

**Comment by PoignardAzur** [comment]
I think we can close this now that render tests work?

**Pull request closed without merging**